### DISCUSSION TITLE
Feature Idea / Request

### DISCUSSION THREAD

**Discussion by TordanDeDanann** [LocuraDU/DU-DamageReport#2] [Ideas]
I might get around to forking this myself, but...
I would like to be able to set a new color and flag for items that are on their last life.

My expected use would be having items on their last life stay at the bottom of  the HUD window list in Blue.

**Comment by CredenceHamby** [comment]
A complete rewrite of the script is on the way. Will see if I have a way of adding something like this in. The issue is I don't know if  we have a way to pull number of lives from the code we use from NQ. Ill have to look into it. 

Currently no ETA on new version just when i get time.
